packages/tools/cookbook/contracts/coin.module.pact

```diff
@@ -0,0 +1,681 @@
+(module coin GOVERNANCE
```

Comment:
I don't mind including this file but maybe we can add a line in the README about checking the canonical repo for any coin contract upgrades - if/when we go to coin v6 we may forget to update this

Reply:
Actually: is this included just for reference or is it required by code?

Reply:
Oh cool, wasn't sure if it was there for users' reference